### ISSUE TITLE
Custom project URLs

### ISSUE THREAD

**Issue by probablykasper** [sdispater/poetry#466]
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.

## Issue
Feature request: Add support in pyproject.toml for specifying custom project URLs.

In flit, it's done like this:
```python
[tool.flit.metadata.urls]
Documentation = "https://flit.readthedocs.io/en/latest/"
```

[Twine](https://pypi.org/project/twine/) is an example of a package that specifies a custom URLs. Twine's setup.py looks like this:
```
[...]
setup(
    [...]
    project_urls={
        'Packaging tutorial': 'https://packaging.python.org/tutorials/distributing-packages/',
        'Twine documentation': 'https://twine.readthedocs.io/en/latest/',
        'Twine source': 'https://github.com/pypa/twine/',
    },
    [...]
)
```

**Comment by chrahunt** [comment]
These fields correspond to the [`Project-URL`](https://packaging.python.org/specifications/core-metadata/#project-url-multiple-use) metadata field. The setuptools documentation can also be found [here](https://setuptools.readthedocs.io/en/latest/setuptools.html).

**Comment by timonbimon** [comment]
I think this issue has been resolved with #1137 :)

**Comment by probablykasper** [comment]
Seems like it's fixed
